### PR TITLE
Changed font style

### PR DESCRIPTION
**Pull request by DivyanshiNarang**
Resolves issue #505 

Changed font style to maintain the consistency of the page.

Before: 
Screenshot 2023-06-24 235652

Now:
Screenshot 2023-06-24 235156

### CODE REVIEW

**Comment by gurjeetsinghvirdee** [comment]
I mean really you are assigning font fixing issues @narayansoni 

Definitely, I will check your repo PRs day after tomorrow.

BcfdMonkeyGIF

**Comment by narayan954** [comment]
> I mean really you are assigning font fixing issues @narayan954 
> 
> Definitely, I will check your repo PRs day after tomorrow.
> 
> BcfdMonkeyGIF
> 

that's what I mentioned, I'll no longer allow single line issues.

File: src/components/Navbar/index.jsx [hunk 1]
```diff
@@ -89,7 +89,7 @@ function Navbar({ onClick, open, setOpen, user, setUser, setLogout }) {
                       className="nav-menu-item"
                       onClick={() => navigate("/dummygram/favourites")}
                     >
-                      <Typography fontFamily="serif" fontSize="1rem">
+                      <Typography fontFamily="Poppins" fontSize="1rem">
```
Comment (narayan954) [inline src/components/Navbar/index.jsx:92]: removing the attribute fontFamily should've worked as it is inherited from the root/index... If it doesn't, means some other component with other property exists in between and needs to be fixed as well. Can you please check and confirm?

**Review by narayan954 — CHANGES_REQUESTED** [review]
check comment

File: src/components/Navbar/index.jsx (hunk 1 above)
Comment (DivyanshiNarang) [reply src/components/Navbar/index.jsx:92]: Yeah sure, I will check this. @narayan954

Comment (DivyanshiNarang) [reply src/components/Navbar/index.jsx:92]: Hello @narayan954 . Yeah you're right, removing the attribute fontFamily worked and solved this issue. Should I commit the changes?

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Pull request merged**